docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,607 @@
+Alice receives the certificate that specifies the roles assigned to her in the root cluster. This metadata
+about her roles is contained in the certificate extensions and is protected by the signature of the root 
+cluster certificate authority so it can't be tampered with.
 
-```code
-$ tctl tokens rm (=presets.tokens.first=)
-```
+When Alice connects to a resource in the leaf cluster `leaf-east.example.com`, she's identified as an external user
+with a certificate signed by an external certificate authority. Based on the role mapping rules in the leaf
+cluster, Alice is assigned the `stage-access` role. This role allows her to access `mongodb.stage.example.com`
+but not `mongodb.prod.example.com`.
 
-</Details>
+| Role in the root cluster | Mapped role in the leaf cluster |
+| ------------------------ | ------------------------------- |
+| full-access | stage-access |
 
-<Notice
-  type="tip"
->
+In this example, the leaf cluster `leaf-east.example.com` denies Alice access to the `mongodb.prod.example.com` resource
+because her `full-access` role in the root cluster is mapped to the `stage-access` role in this leaf cluster.
+With role mapping, leaf cluster administrators can control the privileges external users are granted.
+Although role mapping can be as simple as assigning users to the same roles in the root and leaf clusters,
+you can also use mapping to downgrade a user's privileges or restrict access to specific resources.
 
-  It's important to note that join tokens are only used to establish a
-  connection for the first time. Clusters will exchange certificates and
-  won't use tokens to re-establish their connection afterward.
+Now that you know what a trusted cluster is and how it works, you can use this guide 
+to learn how to:
 
-</Notice>
+- Identify root and leaf clusters.
+- Add trusted cluster resources.
+- Generate an invitation token to establish a trust relationship between a root and leaf cluster.
+- Establish permission mapping between clusters using Teleport roles.
+- Enable and disable trust between clusters.
 
-### Define a Trusted Cluster resource
+## Prerequisites
 
-On your local machine, create a file called `trusted_cluster.yaml` with the
-following content:
+To complete the steps in this guide, verify your environment meets the following requirements:
 
-```yaml
-# trusted_cluster.yaml
-kind: trusted_cluster
-version: v2
-metadata:
-  name: rootcluster.example.com
-spec:
-  enabled: true
-  token: (=presets.tokens.first=)
-  tunnel_addr: rootcluster.example.com:11106
-  web_proxy_addr: rootcluster.example.com:443
-  role_map:
-    - remote: "access"
-      local: ["visitor"]
-```
+- Access to **two** Teleport cluster instances.
 
-Change the fields of `trusted_cluster.yaml` as follows:
+  - Teleport Community Edition or Teleport Enterprise if you manage the Teleport clusters on your own network.
+  - Teleport Team or Teleport Enterprise Cloud if you access Teleport as a managed cloud-based service.
```

Comment:
I think the wording here is slightly off. To me, this sounds like you would access Teleport as a managed cloud-based service and, based on that, use Teleport Team or Teleport Enterprise Cloud. But Teleport Team/Enterprise Cloud _are_ managed cloud-based services.

Reply:
I don't think this adds anything. Removing it.